include/mbedtls/config_psa.h

```diff
@@ -40,6 +40,20 @@ extern "C" {
 
 #if defined(MBEDTLS_PSA_CRYPTO_CONFIG)
 
+#if defined(PSA_WANT_ALG_CCM)
+#if !defined(MBEDTLS_PSA_ACCEL_ALG_CCM)
+#define MBEDTLS_PSA_BUILTIN_ALG_CCM 1
+#define MBEDTLS_CCM_C
+#endif /* !MBEDTLS_PSA_ACCEL_ALG_CCM */
+#endif /* PSA_WANT_ALG_CCM */
+
+#if defined(PSA_WANT_ALG_CMAC)
+#if !defined(MBEDTLS_PSA_ACCEL_ALG_CMAC)
```

Comment:
We also want to declare these if `PSA_HAVE_SOFT_BLOCK_CIPHER` is declared, so please move CMAC to after `PSA_WANT_ALG_STREAM_CIPHER`, and add this as a conditional.

We also want to declare `PSA_HAVE_SOFT_BLOCK_MODE` if `defined(PSA_WANT_ALG_CMAC)
&& !defined(MBEDTLS_PSA_ACCEL_ALG_CMAC)` is true, so please add it to the ifdef that declares it.